recipes/glaze/all/conanfile.py

```diff
@@ -51,6 +51,13 @@ def validate(self):
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
+    def build(self):
+        if Version(self.version) >= "1.9.7":
+            replace_in_file(self, os.path.join(self.source_folder, "include", "glaze", "csv", "write.hpp"),
```

Comment:
Hello @toge! It seems like this patch is related to the msvc failure: https://github.com/conan-io/conan-center-index/pull/21748#issuecomment-1859597357 

It looks weird that error only happening in CCI, but taking a look on https://github.com/stephenberry/glaze/issues, I see more related issues not to std:;decay, but other C++ features. 

I would request sending an issue to discuss with the upstream first, or sending a PR with that fix. 

Regards.

Reply:
I agree with you.
It seems to be a msvc bug.
It is almost same as following. 
https://developercommunity.visualstudio.com/t/164-regression-MSVC-1424-gets-intern/848410?q=std%3A%3Adecay_t.

Reply:
@uilianries 
Fixed.

Reply:
@toge, thank you for pointing it!